code/modules/admin/verbs/freeze.dm

```diff
@@ -26,32 +26,33 @@ var/global/list/frozen_mob_list = list()
 /mob/living/var/frozen = null //used for preventing attacks on admin-frozen mobs
 /mob/living/var/admin_prev_sleeping = 0 //used for keeping track of previous sleeping value with admin freeze
 
-/mob/living/proc/admin_Freeze(var/client/admin, skip_overlays = FALSE)
+/mob/living/proc/admin_Freeze(client/admin, skip_overlays = FALSE)
 	if(istype(admin))
 		to_chat(src, "<b><font color= red>You have been frozen by [admin]</b></font>")
 		message_admins("<span class='notice'>[key_name_admin(admin)]</span> froze [key_name_admin(src)]")
 		log_admin("[key_name(admin)] froze [key_name(src)]")
 
 	var/obj/effect/overlay/adminoverlay/AO = new
 	if(skip_overlays)
-		src.overlays += AO
+		overlays += AO
 
-	anchored = 1
```

Comment:
Can you explain why you're removing this? (though an anchored mob does seem really weird)

Reply:
Yeah, good point I should explain. Unless I'm mistaken, I found that turning a mob's anchored to 1 doesn't actually stop them from moving. I replaced this with `canmove = 0/1` which actually stops them from moving.

Reply:
Maybe it stops them from being moved by other things, ie space wind? But it's possible that it's really just useless, true.

Reply:
Yeah so you're right, anchored prevents the anchored mob from getting pulled/moved by other things, so adding that back in.